### PR TITLE
Update README.md and point to linaro-mmwg meta-freescale fork

### PR DESCRIPTION
**Pull request by petegriffin**

### CODE REVIEW

File: cmf-thud-freescale.xml [hunk 1]
```diff
@@ -30,5 +30,5 @@
     <project name="meta-gplv2" path="meta-gplv2" revision="thud" remote="yocto-cgit"/>
     <project name="meta-rust/meta-rust" path="meta-rust" revision="master" remote="github"/>
 
-    <project name="andrey-konovalov/meta-freescale" path="meta-freescale" remote="github" revision="mmwg-master" />
+    <project name="linaro-mmwg/meta-freescale" path="meta-freescale" remote="github" revision="mmwg-master" />
```
Comment (psivasubramanian) [inline cmf-thud-freescale.xml:33]: Maybe we will stick to initial revision of linaro-mmwg/meta-freescale (same as now) unless warrior patches doesn't cause any issue.

Comment (petegriffin) [reply cmf-thud-freescale.xml:33]: it is mainly just backported gcc8 fixes for the imx op-tee port to enable it to compile OK, so I think it should be OK

Comment (psivasubramanian) [reply cmf-thud-freescale.xml:33]: oh ok.

**Review by psivasubramanian — APPROVED** [review]

File: cmf-thud-freescale.xml (hunk 1 above)
Comment (moorthy-bs) [reply cmf-thud-freescale.xml:33]: LGTM. Build and image verified

**Pull request merged**